### PR TITLE
Problem: memiavl configs are not defined in app.toml

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- integrate custom configs into app.toml

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,7 @@
 - [#950](https://github.com/crypto-org-chain/cronos/pull/950) Implement memiavl and integrate with state machine.
 - [#985](https://github.com/crypto-org-chain/cronos/pull/985) Fix versiondb verify command on older versions
 - [#998](https://github.com/crypto-org-chain/cronos/pull/998) Bump grocksdb to v1.7.16 and rocksdb to v7.10.2
+- [#]() Add memiavl configs into app.toml
```
Comment (yihuang) [inline CHANGELOG.md:34]:
```suggestion
- [#1028](https://github.com/crypto-org-chain/cronos/pull/1028) Add memiavl configs into app.toml
```

File: app/app.go [hunk 2]
```diff
@@ -353,7 +352,7 @@ func New(
 		// cms must be overridden before the other options, because they may use the cms,
 		// FIXME we are assuming the cms won't be overridden by the other options, but we can't be sure.
 		cms := rootmulti.NewStore(filepath.Join(homePath, "data", "memiavl.db"), logger)
-		cms.SetAsyncWAL(cast.ToBool(appOpts.Get(FlagAsyncWAL)))
+		cms.SetAsyncWAL(cast.ToBool(appOpts.Get(FlagAsyncCommit)))
```
Comment (mmsqe) [inline app/app.go:355]: not sure why set SetAsyncWAL with FlagAsyncCommit

File: app/app.go [hunk 3]
```diff
@@ -151,10 +151,9 @@ const (
 	// NOTE: In the SDK, the default value is 255.
 	AddrLen = 20
 
-	FlagMemIAVL  = "store.memiavl"
-	FlagAsyncWAL = "store.memiavl-async-wal"
```
Comment (mmsqe) [inline app/app.go:155]: do we still need FlagAsyncWAL?

Comment (yihuang) [reply app/app.go:155]: yes, it has to be like this in app setup code, I moved to separate module to look better

**Review by mmsqe — APPROVED** [review]

File: app/app.go (hunk 2 above)
Comment (yihuang) [reply app/app.go]: Good catch, I'll change

**Pull request merged**